MolecularNodes/pkg.py

```diff
@@ -32,52 +32,58 @@ def verify():
     verify_user_sitepackages(site.getusersitepackages())
 
 
-def run_pip(cmd, mirror='', timeout=600):
+def run_pip(cmd_list=[], mirror='', timeout=600):
```

Comment:
Just taking a quick glance through open PRs. I haven't looked too deeply into this PR, or it's purpose, but I will advise against this particular line. Empty lists should never be put as default arguments in Python  - see https://nikos7am.com/posts/mutable-default-arguments/

The preferred pattern is `cmd_list=None` and setting `cmd_list = []` within the function if `cmd_list` is None.

Reply:
Wow thanks! That is quite interesting feature. I will fix this issue with adding explicit typing.

Infact this PR was expected to check whether the installed dependencies meet the version of them recorded in requirements. If no, the installation button will re-appear so that users can upgrade them in an easier way.

Reply:
Thanks for the info @janash ! The expert advice is definitely welcomed :)